src/libecalc/fixtures/cases/all_energy_usage_models/all_models_dto.py

```diff
@@ -990,7 +990,7 @@ def methane_venting(regularity) -> YamlVentingEmitter:
             YamlVentingEmission(
                 name="CH4",
                 rate=YamlEmissionRate(
-                    value="FLARE;METHANE_RATE", unit=Unit.KILO_PER_DAY.name, type=RateType.STREAM_DAY
+                    value="FLARE;METHANE_RATE", unit=YamlEmissionRateUnits.KILO_PER_DAY, type=RateType.STREAM_DAY
```

Comment:
If we are to have different enums/strings for different types of equipment, there will be a lot of duplication and it will also not be a "general way" to deal with units in eCalc, which I do not agree with. Instead we should possibly support different ways of "writing" the unit, and that it should be the same all over the place, instead of KG_PR_DAY one place and KILOGRAMS_PER_DAY another place ...lets discuss